Userland/Libraries/LibGfx/ImageFormats/GIFWriter.cpp

```diff
@@ -193,6 +223,10 @@ ErrorOr<NonnullOwnPtr<AnimationWriter>> GIFWriter::start_encoding_animation(Seek
 
     BigEndianOutputBitStream bit_stream { MaybeOwned<Stream> { stream } };
     TRY(write_logical_descriptor(bit_stream, dimensions));
+
+    // Vendor extension to support looping
+    TRY(write_netscape_extension(bit_stream));
```

Comment:
`WebPWriter::start_encoding_animation()` takes a loop_count parameter – do you want to add a parameter too?

Reply:
can do!